pkg/config/loader_test.go

```diff
@@ -562,6 +563,7 @@ multizone:
       maxMsgSize: 1
       msgSendTimeout: 10s
       nackBackoff: 11s
+      disableSOTW: false
```

Comment:
won't this fail in unit tests because it's the default?

Reply:
Right, please use "inverse" in loader_test.go to see if we actually can set this value